generated/nidaqmx/_task_modules/channels/channel.py

```diff
@@ -198,102 +198,43 @@ def chan_type(self):
         :class:`nidaqmx.constants.ChannelType`: Indicates the type of
             the virtual channel.
         """
-        val = ctypes.c_int()
 
-        cfunc = lib_importer.windll.DAQmxGetChanType
-        if cfunc.argtypes is None:
-            with cfunc.arglock:
-                if cfunc.argtypes is None:
-                    cfunc.argtypes = [
-                        lib_importer.task_handle, ctypes_byte_str,
-                        ctypes.POINTER(ctypes.c_int)]
 
-        error_code = cfunc(
-            self._handle, self._name, ctypes.byref(val))
-        check_for_error(error_code)
-
-        return ChannelType(val.value)
+        val = self._interpreter.get_chan_attribute_int32(
+                self._handle, self._name, 6271)
```

Comment:
We most commonly refer to attribute IDs by their hex value, so 0x187F here. Do you mind updating that? Bonus points if we created constants for all of these 😲

Reply:
Constants ala the C header

```c
#define DAQmx_ChanType                                                   0x187F // Indicates the type of the virtual channel.
```

Reply:
Sure. Updated.